src/views/Register/index.jsx

```diff
@@ -6,10 +6,12 @@ import NameForm from "./NameForm";
 const Register = () => {
 	const [status, setStatus] = useState({ state: "none", message: "" });
 	const [currentIndex, setCurrentIndex] = useState(0);
-	const form_count = 5;
+
+	// Members are scalable using the default value. (5 forms, with the first one being team name and team count.)
+	const [memberCount, setMemberCount] = useState(4);
 
 	const [submitFunctions, setSubmitFunctions] = useState({});
-	const [teamInfo, setTeamInfo] = useState({});
+	const [, setTeamInfo] = useState({});
```

Comment:
If `setTeamInfo` is not using, please remove it from the code.

Reply:
Acctually to manage the state, I need it, if you note the setTeamInfo function in line 43, I'm using it to get the previous state in the code, but I'm not using the acctual teamInfo variable in the code.